tests/acceptance/run.sh

```diff
@@ -264,7 +264,7 @@ function run_behat_tests() {
 	# Thanks to https://en.wikipedia.org/wiki/Tee_(command) and
 	# https://stackoverflow.com/questions/23416278/how-to-strip-ansi-escape-sequences-from-a-variable
 	# for ideas.
-	FAILED_SCENARIO_PATHS=$(echo "${FAILED_SCENARIO_PATHS_COLORED}" | sed "s/\x1b[^m]*m//g")
+	FAILED_SCENARIO_PATHS=$(echo "${FAILED_SCENARIO_PATHS_COLORED}" | sed "s/\x1b\[[0-9;]*m//g" | sed 's/\(\.feature:[0-9]\+\).*/\1/')
```

Comment:
not sure why you have changed the first regex, but I think we can make the second easier to read and the whole thing easier to understand
```suggestion
    FAILED_SCENARIO_PATHS=$(echo "${FAILED_SCENARIO_PATHS_COLORED}" | sed "s/\x1b[^m]*m//g")

    # some comment that explains what happens here, because the above comment only talks about the color codes
	FAILED_SCENARIO_PATHS=$(echo "${FAILED_SCENARIO_PATHS}" | sed 's/\(on line \d+\)//')
```